packages/react-examples/.storybook/preview.js

```diff
@@ -161,28 +177,7 @@ function generateStoriesFromExamples(key, stories, req) {
     return;
   }
 
-  /** @type {string} */
-  let componentName;
-
-  // Story URLs are generated based off the story name
-  // In the case of `react-components` a (package name) suffix is added to each story
-  // This results in a difference name and URL between individual storybooks and the react-components suite storyboo
-  // https://storybook.js.org/docs/react/configure/sidebar-and-urls#permalinking-to-stories
-  // Use the id property in stories to ensure the same URL between individual and suite storyboo
```

Comment:
Do you mind if this still gets reflected smwhere ? either in `generateComponentName` or a few lines below where `id` is assigned to the story

Reply:
ah i can't read :)